### PR TITLE
SITES-568 Fix codeclimate badges

### PR DESCRIPTION
**Pull request by bjgordon**

### CODE REVIEW

File: app/controllers/admin/devise/registration_controller.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module Admin
+
+
+  class RegistrationController < Devise::RegistrationsController
+    def create
+      # do your checks
+      super
+    end
+  end
+end
```
Comment (tessereth) [inline app/controllers/admin/devise/registration_controller.rb]: Is this supposed to be in this PR?

Comment (bjgordon) [inline app/controllers/admin/devise/registration_controller.rb]: Blah I dont know where that came from!

**Comment by tessereth** [comment]
LGTM

**Comment by ltankey** [comment]
LGTM

**Pull request merged**